heroic-component/src/main/java/com/spotify/heroic/common/Feature.java

```diff
@@ -53,7 +53,12 @@
      * This will assert that there are data outside of the range queried for, which is a useful
      * feature when using a dashboarding system.
      */
-    SHIFT_RANGE("com.spotify.heroic.shift_range");
+    SHIFT_RANGE("com.spotify.heroic.shift_range"),
+
+    /**
+     * Enable feature to cause data to be fetched in slices.
```

Comment:
Describe in more detail what "fetching in slices" mean?
It's also worth mentioning that the benefit of this feature is backend-dependent.